src/sentry/web/urls.py

```diff
@@ -472,6 +472,12 @@
     url(r"^join-request/", react_page_view, name="join-request"),
     # User Feedback
     url(r"^user-feedback/", react_page_view, name="user-feedback"),
+    # Project on-boarding
+    url(
+        r"^(?P<project_slug>[\w_-]+)/getting-started",
```

Comment:
I don't think we have reserved all the project slugs that would overlap with our top-level routes.